src/renderer/components/server-modal-form.jsx

```diff
@@ -314,11 +314,22 @@ export default class ServerModalForm extends Component {
           </div>
           <div className={`four wide field ${this.highlightError('database')}`}>
             <label>Database/Keyspace</label>
-            <input type="text"
-              name="database"
-              placeholder="Database"
-              value={this.state.database || ''}
-              onChange={::this.handleChange} />
+            <div className="ui action input">
```

Comment:
Could you make the input file button only be visible for sqlite? I believe this button could be confuse for most users using the other database clients that does work with local files.